### ISSUE TITLE
Question - webpack upgrade

### ISSUE THREAD

**Issue by davidcsally** [PubMatic/OpenWrap#178]
Hello! I see that this project uses webpack 1 to bundle. Have you looked into updating to the latest webpack version for further bundle optimization?

**Comment by pm-shashank-jain** [comment]
Yes @davidcsally, it is scheduled to update the all the dependency package along with webpack
